src/Apps/Artwork/Components/ImageCarousel.tsx

```diff
@@ -198,11 +198,17 @@ export class ImageCarousel extends React.Component<
     const xs = breakpoint === "xs"
     const Image = xs ? SmallImage : LargeImage
 
-    return (
-      <Lightbox deepZoom={this.image.deepZoom} enabled={this.image.is_zoomable}>
-        <Image src={this.image.uri} />
-      </Lightbox>
-    )
+    if (typeof window !== "undefined") {
```

Comment:
So far as I'm aware, this is going to cause a rehydration miss. To avoid that we'd have to only enabling rendering this on componentDidMount. Speaking of, it'd be nice just to have a `ClientOnly` component or something that we could wrap these things in. 

/cc @alloy

Reply:
Yeah this should be done on mount